src/main/java/androidx/text/Html.kt

```diff
@@ -31,9 +34,11 @@ import android.text.Spanned
  * @param flags Additional option to set the behavior of the HTML parsing. Default is set to
  * [Html.FROM_HTML_MODE_LEGACY] which was introduced in Android 7.0.
  */
-fun CharSequence.parseAsHtml(flags: Int? = 0): Spanned {
-    return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-        Html.fromHtml(toString(), flags ?: Html.FROM_HTML_MODE_LEGACY)
+fun CharSequence.parseAsHtml(
+    @SuppressLint("InlinedApi") flags: Int = FROM_HTML_MODE_LEGACY
```

Comment:
Why aren't we using 0 here? If someone wants legacy parsing they should be the ones who have to specify it.

Reply:
So weird.